.github/workflows/test.yml

```diff
@@ -0,0 +1,292 @@
+        MATRIX_EVAL: "CC=gcc-5 && CXX=g++-5"
+      run: |
+        source .github/travis/common.sh
+        ./.github/travis/setup.sh
+        ./.github/travis/build.sh
+        ./.github/travis/setup.sh
+
+
+  Regression:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          {
+            name: 'Basic',
+            params: '-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on',
+            suite: 'vtr_reg_basic'
+          },
+          {
+            name: 'Basic with NO_GRAPHICS',
+            params: '-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on -DVPR_USE_EZGL=off',
+            suite: 'vtr_reg_basic'
+          },
+          {
+            name: 'Basic with VTR_ENABLE_DEBUG_LOGGING',
+            params: '-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on -DVTR_ENABLE_DEBUG_LOGGING=on',
+            suite: 'vtr_reg_basic'
+          },
+          {
+            name: 'Strong',
+            params: '-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on',
+            suite: 'vtr_reg_strong'
+          },
+          {
+            name: 'Valgrind Memory',
+            params: '-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on',
+            suite: 'vtr_reg_valgrind_small'
+          }
+        ]
+    name: 'R: ${{ matrix.name }}'
+    steps:
+
+    - uses: actions/checkout@v2
+    - run: ./.github/gha/install_dependencies.sh
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - run: pip install -r requirements.txt
+
+    - name: Test
+      env:
+        CMAKE_PARAMS: ${{ matrix.params }}
+        MATRIX_EVAL: "CC=gcc-5 && CXX=g++-5"
+      run: |
+        source .github/travis/common.sh
+        ./.github/travis/setup.sh
+        ./.github/travis/build.sh
+        ./run_reg_test.py ${{ matrix.suite }} -show_failures -j2
+        ./.github/travis/setup.sh
+
+
+  Sanitized:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - { name: 'Basic',  suite: 'vtr_reg_basic'  }
+        - { name: 'Strong', suite: 'vtr_reg_strong' }
+    name: 'S: ${{ matrix.name }}'
+    steps:
+
+    - uses: actions/checkout@v2
+    - run: ./.github/gha/install_dependencies.sh
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - run: pip install -r requirements.txt
+
+    - name: Test
+      env:
+        CMAKE_PARAMS: '-DVTR_ASSERT_LEVEL=3 -DVTR_ENABLE_SANITIZE=on -DVTR_IPO_BUILD=off -DWITH_BLIFEXPLORER=on'
+        MATRIX_EVAL: "CC=gcc-5 && CXX=g++-5"
+        BUILD_TYPE: debug
+        LSAN_OPTIONS: 'exitcode=42' #Use a non-standard exit code to ensure LSAN errors are detected
+      run: |
+        source .github/travis/common.sh
+        ./.github/travis/setup.sh
+        ./.github/travis/build.sh
+        # We skip QoR since we are only checking for errors in sanitizer runs
+        ./run_reg_test.py ${{ matrix.suite }} -show_failures -j2
```

Comment:
This needs `-skip_qor` because the sanitized version doesn't have good runtime, which is why the basic sanitized version failed.  The vtr_reg_strong takes too long on GH Actions, and so should be dropped.  The Kokoro CI "Strong Sanitized" has ultimited runtime, so I don't think we need the strong sanitized here.

Reply:
FYI - GitHub actions has a significantly longer maximum timeout than Travis.
The default is 360 minutes, but it can be extended to be much longer by using https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes

 * Job execution time - Each job in a workflow can run for up to 6 hours of execution time. If a job reaches this limit, the job is terminated and fails to complete.
 * Workflow run time - Each workflow run is limited to 72 hours. If a workflow run reaches this limit, the workflow run is cancelled.

Reply:
In fact, `Regression · Strong · vtr_reg_strong` run rather fast: ~30min https://github.com/umarcor/vtr-verilog-to-routing/runs/1534764902?check_suite_focus=true. Timeout happened with `Sanitized · Strong · vtr_reg_strong`. Let's see with `-skip_qor` now.